### DISCUSSION TITLE
How do I exclude bots from changelogs?

### DISCUSSION THREAD

**Discussion by hasparus** [intuit/auto#1975] [Q&A] [answered]
Howdy :wave:

Sorry for a noob question but `exclude` list in all contributors plugin doesn't affect _Authors_ section in the changelog, right?
Dependabot is definitely the most active contributor in my library, and I keep removing him from changelogs.

Should I use [`omitCommit`](https://intuit.github.io/auto/docs/plugins/log-parse-hooks#omitcommit) for this?

I've found the [bot-list](https://github.com/intuit/auto/blob/d665c3a44f55d547aac9749e2c95576305121cf1/packages/bot-list/src/index.ts) which is ignored in `core`, but it only applies to additional release notes if I'm not wrong.

**Comment by hipstersmoothie** [comment]
Hey do you have a public repo available to view? Dependabot like to change it's name a lot, probably only need to add the new name to the bot list

**Reply by hasparus** [reply]
Here's my repo and auto config — https://github.com/system-ui/theme-ui/blob/develop/auto.config.ts

I tend to remove everything from Dependabot from changelogs, as there are [tons of them](https://github.com/system-ui/theme-ui/commit/b0552e962efd5495ed8742357ed1a3e7e9af0a24#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edL33-L55) and they don't affect the public API (and if they do, a human needs to write some migration notes anyway).

**Comment by hasparus** [comment, marked as answer]
Found [omit-commits plugin](https://intuit.github.io/auto/docs/generated/omit-commits#installation) and ignored all `chore(` commits.

**Reply by hipstersmoothie** [reply]
Glad that worked for you! I really wish dependabot would stop changing it's name lol
